### PR TITLE
Feat: Adds tokens to `Alert`

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
This PR applies the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to `Alert`: new scoped tokens linked to global ones.

## How does it work?

This PR uses local variables to parameterize the `Alert` properties, then connects these scoped tokens to the global ones.

### New local tokens for `Alert`

Screen Shot 2022-05-30 at 19 39 21

#### Default

|Local token|Default value / global token linked|
|-|-|
|`--fs-alert-height`                       | `var(--fs-spacing-7)`|
|`--fs-alert-padding-left`                 | `var(--fs-spacing-3)`|
|`--fs-alert-padding-right`                | `var(--fs-alert-padding-left)`|
|||
|`--fs-alert-bkg-color`                    | `var(--fs-color-highlighted-bkg)`|
|||
|`--fs-alert-text-color`                   | `var(--fs-color-highlighted-text)`|
|`--fs-alert-text-size`                    | `var(--fs-text-size-1)`|
|||
|`--fs-alert-link-color` | `var(--fs-alert-text-color)`|
|`--fs-alert-link-color-visited` | `var(--fs-alert-text-color)`|
|||
|`--fs-alert-button-text-color` | `var(--fs-alert-text-color)`|
|`--fs-alert-button-bkg-color` | `var(--fs-alert-bkg-color)`|
|`--fs-alert-button-border-radius` | `var(--fs-border-radius)`|

## How to test it?

The `Alert` component and its variants should look just as it was before these changes.
[Preview Link](https://gatsby-store-storybook-git-feat-theming-alert-f5a020-faststore.vercel.app/?path=/docs/molecules-alert--default-story)

## Checklist

- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 
- [x] Updated the Storybook - *if applicable*.
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.

### CODE REVIEW

File: src/components/ui/Alert/Alert.tsx [hunk 1]
```diff
@@ -36,20 +47,27 @@ function Alert({
...
+      <p data-fs-alert-content>{children}</p>
 
       {link && (
-        <div data-alert-link>
-          <Link to={link.to}>{link.text}</Link>
-        </div>
+        <Link data-fs-alert-link to={link.to}>
+          {link.text}
+        </Link>
       )}
```
Comment (eduardoformiga) [inline src/components/ui/Alert/Alert.tsx:64]: Nice!

File: src/components/ui/Alert/Alert.tsx [hunk 2]
```diff
@@ -36,20 +47,27 @@ function Alert({
...
 
       {dismissible && (
-        <Button data-alert-button aria-label="Close" onClick={handleClose}>
-          <Icon name="X" width={18} height={18} weight="bold" />
+        <Button data-fs-alert-button aria-label="Close" onClick={handleClose}>
+          <span>
```
Comment (eduardoformiga) [inline src/components/ui/Alert/Alert.tsx:68]: do we need this span tag here?

**Review by eduardoformiga — APPROVED** [review]
Good Job! 👏

**Review by lucasfp13 — APPROVED** [review]
Niceee! 💯

**Pull request merged**